dockerfiles/init/modules/keycloak/files/che/login/register.ftl

```diff
@@ -0,0 +1,76 @@
+<#import "template.ftl" as layout>
```

Comment:
@mmorhun how do you manage the upgrade of keycloak then ?
Let say they fix an issue in the base template. How the fix is then applied on our template ? it is not.